data-jdbc/src/test/groovy/io/micronaut/data/jdbc/DatabaseTestPropertyProvider.groovy

```diff
@@ -14,6 +14,11 @@ trait DatabaseTestPropertyProvider implements TestPropertyProvider {
         return SchemaGenerate.CREATE
     }
 
+    List<String> packages() {
```

Comment:
Found out that some H2 tests were including Postgres specific entities so this limits to only minimally needed entities for all the tests.